app/views/kyc/organisation-url.njk

```diff
@@ -0,0 +1,46 @@
+{% extends "../layout.njk" %}
+
+{% block pageTitle %}
+  What is your Organisation URL? - {{ currentService.name }} - GOV.UK Pay
+{% endblock %}
+
+{% block mainContent %}
+
+<div class="govuk-grid-column-two-thirds">
+
+  {{ govukBackLink({
+    text: "Back to Your PSP",
+    classes: "govuk-!-margin-top-0",
+    href: formatAccountPathsFor(routes.account.yourPsp.index, currentGatewayAccount.external_id, credential.external_id)
+  }) }}
+
+  <h1 class="govuk-heading-l govuk-!-margin-bottom-6"><label for="vat-number">What is your organisation’s URL?</label></h1>
```

Comment:
It looks like this page might have been missed in the content review. There is a separate story to make sure all the pages are using the finalised content at the end, but might want to raise with Justin that this page needs reviewing. The content document is here https://docs.google.com/document/d/1-_ohNR4DjT26alRaIAdu2cR5Rqq9TEbhGYJEkoFlQkA/edit#heading=h.l61akk9fle6h

Reply:
The content for this page is in that Google doc - under `Add Organisation detail`.
I have updated the content to the laters version as per that Google doc.